source/_components/frontend.markdown

```diff
@@ -76,3 +76,20 @@ automation:
 ### {% linkable_title Manual Theme Selection %}
 
 When themes are enabled in the `configuration.yaml` file, a new option will show up in the Configuration panel under `configuration.yaml` called "Set a theme." You can then choose any installed theme from the dropdown list and it will be applied immediately.
+
+
+## {% linkable_title Loading extra HTML %}
+
+Starting with version 0.53 you can specify extra html files to load:
```

Comment:
**Spelling & Grammar**:
- The sentence ends with a `:`, but is not follow by something, it should end with a `.`
- `html` -> `HTML`

Reply:
Done